remoto/log.py

```diff
@@ -12,7 +12,9 @@ def reporting(conn, result, timeout=None):
         try:
             received = result.receive(timeout)
             level_received, message = list(received.items())[0]
-            log_map[level_received](message.strip('\n'))
+            message = message.splitlines()
```

Comment:
this doesn't look right, `message` is a single line already. If you want to remove the `^M` char just add it to the `strip()` call:

    message.strip('\n^M')